package.json

```diff
@@ -6,10 +6,13 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "i18next": "^19.8.2",
+    "i18next-browser-languagedetector": "^6.0.1",
     "node-sass": "^4.14.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-hook-form": "^6.9.4",
+    "react-i18next": "^11.7.3",
```

Comment:
its better for you to work directly with i18next alone, because with thisl ibrary you have to work with hooks, and that  blocks you to work with translation on utils or constants without passing the `t` function to those functions/constants